### ISSUE TITLE
time to run RTSC.bat

### ISSUE THREAD

**Issue by zimbalam** [Batlez/HiddenVM#5]
![Screenshot 2024-05-20 180046](https://github.com/Batlez/HiddenVM/assets/170376251/e2c8d647-1581-4d40-ac91-500c6a2c08a7)

**Comment by Batlez** [comment]
That is not an error. It is suppose to detect if it's running or not, and then close it. You would run RTSC after creating your virtual machine. Then you would put in the location of where its located, and the name of it. It would be something like this:

![324279708-93b09350-86ad-4fb9-966c-ed1f56a12f1e](https://github.com/Batlez/HiddenVM/assets/63690709/04eaf6bb-6b9a-49f4-a739-6a41ffc57c1b)

**Comment by zimbalam** [comment]
After running RTSC, I will start launching windows, right?

**Comment by Batlez** [comment]
Yes after you run the batch file, you would then start up the virtual machine.

**Comment by Batlez** [comment]
Please try out the new version that came out and let me know if you still have issues!
